blocks/columns/columns.js

```diff
@@ -1,3 +1,25 @@
+function addSlimPromoClick(block, mq) {
```

Comment:
IMO, it should not collapse the banner if you click it again, or click any link in it. But it does.

Reply:
I can probably make the event scope more tight.